iquise/utils.py

```diff
@@ -0,0 +1,43 @@
+import hashlib, zlib
+import cPickle as pickle
+import urllib
+
+from datetime import timedelta
+
+from django.conf import settings
+from django.utils import timezone
+from django.urls import reverse
+
+from website.models import IQUISE, Donor
+
+def encode_data(data):
+    """Turn `data` into a hash and an encoded string, suitable for use with `decode_data`."""
```

Comment:
what does """ do?

Reply:
To answer a question related to your question: This is the standard python syntax for docstrings (here it is the docstring for the function `encode_data`). A docstring is used when you call python's `help` function. In this case `help(encode_data)` would return that string. Auto documentation generators (like Sphinx) will also use these to generate your nicely formatted docs.

To answer your question: """ is a way to initiate a python string that ignores linebreaks. So the following is illegal python and would give you an error:
```python
foo = "foo
bar"
```
Whereas this is totally fine:
```python
foo = """foo
bar"""
```
It has the added benefit of allowing to use single and double quotes inside a string too without having to escape them.